src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write code here
+const headers = document.querySelectorAll('th');
+const tableBody = document.querySelector('tbody');
+const allEmployees = tableBody.rows;
+
+const sortList = (ulNode, index) => {
+  [...allEmployees].sort((prev, next) => {
+    if (prev.children[index].textContent.charAt(0) === '$') {
+      return convertToNumber(prev.children[index].textContent)
+        - convertToNumber(next.children[index].textContent);
+    } else {
+      return (prev.children[index].textContent)
+        .localeCompare(next.children[index].textContent);
+    }
+  }).forEach(item => ulNode.append(item));
+};
```

Comment:
For readibility I would suggest creating const, also when chaining methods you can start each method from a newline.
Also else statement is not needed
```suggestion
const sortList = (ulNode, index) => {
  [...allEmployees]
    .sort((prev, next) => {
      const prevContent = prev.children[index].textContent;
      const nextContent = next.children[index].textContent;

      if (prevContent.charAt(0) === '$') {
        return convertToNumber(prevContent)- convertToNumber(nextContent);
      } else {
        return (prevContent).localeCompare(nextContent);
      }
    })
  .forEach(item => tableBody.append(item));
```